patches/fix-filter-force-update.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterValue.tsx b/superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterValue.tsx
+index 2817aeb..2e626636c7 100644
+--- a/superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterValue.tsx
++++ b/superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterValue.tsx
+@@ -167,7 +167,7 @@ const FilterValue: React.FC<FilterControlProps> = ({
+      setIsRefreshing(true);
+      getChartDataRequest({
+        formData: newFormData,
+-       force: false,
++        force: shouldRefresh,
```

Comment:
is there an extra space here?

Reply:
I don't fully understand why, as looking at it there seems to be an extra space but removing that space gives the below which appears to be incorrectly indented: 
```
ERROR in src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterValue.tsx:167:8
2024-08-06 09:50:12.615 :: 2024-08-06 09:50:10.261 :: prettier/prettier: Insert `·`
2024-08-06 09:50:12.615 :: 2024-08-06 09:50:10.261 ::     165 |       getChartDataRequest({
2024-08-06 09:50:12.615 :: 2024-08-06 09:50:10.262 ::     166 |         formData: newFormData,
2024-08-06 09:50:12.615 :: 2024-08-06 09:50:10.262 ::   > 167 |        force: shouldRefresh,
2024-08-06 09:50:12.615 :: 2024-08-06 09:50:10.262 ::         |        ^
```